### PR TITLE
Bootstrap for Patroni >= 1.0

### PR DESCRIPTION
**Pull request by ikitiki**

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
Config migration utility added. Reviews are appreciated.

**Comment by ikitiki** [comment]
👍

File: postgres-appliance/convert_config.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+keys_add = {
+    "bootstrap.users.admin.options": ["createrole", "createdb"],
+    "bootstrap.dcs.postgresql.use_pg_rewind": True,
+    "bootstrap.dcs.postgresql.use_slots": True,
+    "bootstrap.dcs.retry_timeout": 10
+}
+
+keys_copy = {
+    "bootstrap.dcs.postgresql.authentication.replication": "postgresql.replication",
```
Comment (alexeyklyukin) [inline postgres-appliance/convert_config.py]: superuser and replication keys are not taken from bootstrap section, but from the postgresql.authentication

**Comment by feikesteenbergen** [comment]
Tested it by running a new Spilo with Patroni 1.0 as well as migrating an existing Spilo from 0.9 to 1.0.
I did not test the migration utility as it does not seem necessary for Spilo: Spilo generates the configuration based upon its environment.
Adding the migration tool to Patroni would be OK I suppose

**Comment by feikesteenbergen** [comment]
👍

**Comment by feikesteenbergen** [comment]
👍

**Comment by feikesteenbergen** [comment]
+1

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**